src/main/java/fr/techad/edc/httpd/search/ContentSearcher.java

```diff
@@ -46,6 +48,7 @@ public class ContentSearcher extends ContentBase {
 
   public ContentSearcher(WebServerConfig webServerConfig) {
     super(webServerConfig);
+    this.config = webServerConfig;
```

Comment:
ContentSearcher need this property. So store it in ContentBase.

Don't forget to add a getter with protected scope.